### DISCUSSION TITLE
Peer info in the main window

### DISCUSSION THREAD

**Discussion by GaryElshaw** [transmission/transmission#3076] [Ideas]
Because i'm on a Macbook Air, and because my eyes are getting older, i permanently have the peers window in the inspector open. I was wondering if i was alone in doing this?

I can of course change the code locally to increase the font size in the main window which helps, but then i wondered if it might be a function that could be included if it was useful for others. Here's a mockup that uses the peers icon in the main window along side of the pause and reveal in finder icon. I thought the peers icon could be a simple switch to reveal or hide connected peers. What do people think? @ckerr @livings124 @mikedld 

![SCR-20220513-ix0](https://user-images.githubusercontent.com/69029666/168194078-f2fe3dfa-618d-491d-b82e-9d283142a473.png)

Edit: In case me or someone else wants to return to this idea, a much better peer-view icon at 32x32.
![peer-infox32](https://user-images.githubusercontent.com/69029666/168193835-20106360-7daa-46a0-a699-6d3ee875097e.png)

**Comment by Oleg-Chashko** [comment]
_I was wondering if i was alone in doing this?_ 

You are not alone, I have the same problem, I also always look at window peers in the inspector. And if only the icons of the countries next to the IP were....  For some reason it mesmerizes (Hypnotic Eyes) me (I thought I had a mental problem, but I'm glad I'm not alone). The only reason I use Tixati app is because there are so many different charts... It mesmerizes (Hypnotic Eyes) me like the look of a cobra in a cartoon.

![hypnotic-eyes_disney-jungle-book_2079](https://user-images.githubusercontent.com/62497891/172250726-81a3182f-571c-490c-b87b-3ca0c257d4f0.jpg)
